spec/lib/efile/md/md502cr_calculator_spec.rb

```diff
@@ -227,7 +227,17 @@
 
     %w[single married_filing_separately dependent].each do |filing_status|
       context "when filing status is #{filing_status}" do
-        let(:filing_status) { filing_status }
+        if filing_status == "dependent"
+          let(:filing_status) { "single" }
+          let(:claimed_as_dependent) { true}
+        else
+          let(:filing_status) { filing_status }
+          let(:claimed_as_dependent) { false }
+        end
```

Comment:
i'm guessing we'll do something similar for the MD502 calculator tests when we have time to fix them up?

Reply:
Yes, that's my thinking